ethcore/vm/src/schedule.rs

```diff
@@ -113,6 +113,49 @@ pub struct Schedule {
 	pub kill_dust: CleanDustMode,
 	/// Enable EIP-86 rules
 	pub eip86: bool,
+	/// Wasm extra schedule settings
+	pub wasm: WasmCosts,
+}
+
+/// Wasm cost table
+pub struct WasmCosts {
+	/// Arena allocator cost, per byte
+	pub alloc: usize,
```

Comment:
Why `usize`? `usize` should only be used for representing memory area sizes. Hence the name. `u64` should prevent any unexpected issues on 32-bit platforms.

Reply:
And why all regular `Schedule` members is usize?

Reply:
No idea, but I think it needs to be fixed.

Reply:
https://github.com/paritytech/parity/blob/c7ea25227a01aa9d22964f84489d7f16a224ce5c/ethcore/evm/src/evm.rs#L59

oh this is because of that, gas counter uses different size across platforms

i don't use it, so i might stick to u64